lms/djangoapps/courseware/url_helpers.py

```diff
@@ -9,19 +9,23 @@
 from xmodule.modulestore.django import modulestore
 
 
-def get_redirect_url(course_key, usage_key):
+def get_redirect_url(course_key, usage_key, unified_course_view=False):
```

Comment:
The issue is that we don't have the request context in this method at the moment, so the alternative is to pass in the request, which I am hesitant to do more of.